recipes-bsp/firmware-imx/firmware-imx-8_8.18.bb

```diff
@@ -18,6 +18,6 @@ addtask deploy after do_install before do_build
 
 PACKAGE_ARCH = "${MACHINE_SOCARCH}"
 
-COMPATIBLE_MACHINE      = "(mx8-nxp-bsp)"
+COMPATIBLE_MACHINE      = "(mx8-nxp-bsp|mx9-nxp-bsp)"
```

Comment:
With `firmware-imx-9` back, this change needs to be dropped as well.

Reply:
@angolini @otavio 
firmware-imx-9 is integrated to firmware-imx-8 recipe. 
How about also merge firmware-imx-8m into formware-imx-8?

Reply:
@junzhuimx, totally agree. See #1426.